src/clue.js

```diff
@@ -1,14 +1,147 @@
 // ITERATION 1
```

Comment:
**A general tip from now**: Delete the console.log from the file on your final submission. Reason -> The tests don't take into account the console.log when verifying your solutions. They only consider the function.

**The exception** _It's ok to use console.log if the purpose is to be sure that the value you are getting back is correct. Once you have confirmed this, however, you may delete it._